sdk/identity/azure_identity/src/lib.rs

```diff
@@ -60,6 +62,7 @@ use std::borrow::Cow;
 #[derive(Debug, Default, Deserialize)]
 #[serde(default)]
 struct EntraIdErrorResponse {
+    error_codes: Vec<i32>,
```

Comment:
For the `error_description`, mainly, you might rely on borrowing for less memory overhead if you're just trying to pick out a few things. We do that for `ErrorResponseInternal` and you can find more details at <https://serde.rs/lifetimes.html#borrowing-data-in-a-derived-impl>.

Reply:
#3322